src/views/console/index.php

```diff
@@ -20,12 +18,16 @@
 foreach ($requirements as $key => $requirement) {
     if ($requirement['condition']) {
         echo $requirement['name'] . ": OK\n";
+        $memo = strip_tags($requirement['memo']);
+        if (!empty($memo)) {
+            echo strip_tags($requirement['memo']) . "\n";
```

Comment:
Why do we need to output it? Also, doing `strip_tags` twice doesn't make sense.

Reply:
Please, remove

Reply:
removed